app/views/locales/_start_a_chapter.en.md

```diff
@@ -0,0 +1,31 @@
+# Anyone Can Start An Awesome Foundation Chapter
+
+The Awesome Foundation isn't actually a foundation at all. Instead, it's a grassroots movement. Our global community is made up of amazing people who have made the decision to make their local communities a bit more awesome. They organize to assemble, launch, and run their own chapters.
+
+That means that you can start an Awesome Foundation chapter right now, in your own community. If you’ve ever dreamed of meeting awesome people you might have never crossed paths with, supporting wild and weird projects in need of just a little boost to become reality, and getting connected to folks all around the world who love doing the same, the Awesome Foundation is for you.
+
+We’re probably biased, but it’s a heckuva lot of fun. Chapters have made amazing things happen all around the world, from building a giant hammock and assembling swings in public places, to helping dogs play poker and letting anyone be Indiana Jones for a day.
+
+### How do you do it?
+
+
+
+Getting a new Awesome Foundation chapter up and running is straightforward – we like keeping things simple.
+
+![Five steps to starting an Awesome Foundation Chapter](/images/new-chapter-steps.png)
+
+The big task is finding ten or so people who will form the core of your chapter. These “trustees” will be the ones funding grants and deciding who to fund, and you’ll be spending a lot of time with them. So, find the most awesome people you can think of and get ‘em on board.
+
+Once you assemble your dream team, we’ll get you linked up with our global community and application management system, and you’ll be up and running! It’s that easy.
+
+Here’s the steps:
+
+1. Check out existing chapters: The Awesome Foundation is in a lot of places, so it’s possible that you might be able to join a chapter that is already rolling in your area. Check the “Chapters” menu above to see if there’s one, and get in touch with them if you see one nearby.
```

Comment:
These render with the number on one line and the text on another -- I'm not sure if there's an easy way to prevent the markup engine from doing that.